### ISSUE TITLE
Uncaught SyntaxError: Unexpected token <

### ISSUE THREAD

**Issue by HanwenG** [VividCortex/angular-recaptcha#156]
Hello,

I am using your stuff but I got this issue. I just download two "js" files in the release and include them in my index.html.

see     <script src="lib/angular/angular-recaptcha.min.js"></script>

But I got a response from the browser as following
Uncaught SyntaxError: Unexpected token <  angular-recaptcha.min.js:5 

Should I include other files from folder "src", such as service.js and directive.js?  If so, can you give me some instruction.

**Comment by TheSharpieOne** [comment]
This usually happens when an HTML page is returned to a JS request. Typically a 404 page or a corporate firewall page. Look inside the dev tools network panel and see what is being returned.

**Comment by HanwenG** [comment]
I think it is because that loading JS issue. Did I forget to include service.js and directive.js?  I just develop something on my local machine.

For the network panel, there is 404 response.

**Comment by TheSharpieOne** [comment]
In the network panel, you should be able to see the JS file which is returned. Does the file look right (something similar to this screenshot below)?
![image](https://cloud.githubusercontent.com/assets/664714/17214094/4f831658-54a5-11e6-9318-e52ce026a0d9.png)

**Comment by HanwenG** [comment]
This is what I got from the response, it seems that the angular-recaptcha.min.js push back some html stuff?

It seems that these js files contain some html?

Any way, I downloaded the js again and it seems loading dont have the problem. I will have a try in my implementation.

Thanks very much.

![image](https://cloud.githubusercontent.com/assets/12438113/17238499/85a30284-559f-11e6-8dcb-a2a6b695b62a.png)
